cmd/av/stack_branch.go

```diff
@@ -236,11 +250,15 @@ func stackBranchMove(
 	tx.DeleteBranch(oldBranch)
 
 	// Finally, actually rename the branch in Git
-	if _, err := repo.Run(&git.RunOpts{
-		Args:      []string{"branch", "-m", newBranch},
-		ExitError: true,
-	}); err != nil {
-		return errors.WrapIff(err, "failed to rename Git branch")
+	if ok, err := repo.DoesBranchExist(oldBranch); err != nil {
+		return err
+	} else if ok {
+		if _, err := repo.Run(&git.RunOpts{
+			Args:      []string{"branch", "-m", newBranch},
+			ExitError: true,
+		}); err != nil {
+			return errors.WrapIff(err, "failed to rename Git branch")
+		}
```

Comment:
Let's add an `else` clause here that prints out a message to the effect of

```
Branch <oldbranch> does not exist in Git. Updating av internal metadata only.
```

Reply:
Done